conf/kubernetes-shuffle-service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: spark-shuffle-service
+    version: 2.1.0
+  name: shuffle
+spec:
+  template:
+    metadata:
+      labels:
+        app: spark-shuffle-service
+    spec:
+      volumes:
+        - name: temp-volume
+          hostPath:
+            path: '/tmp'
```

Comment:
`/tmp` might not be ideal in many setups. e.g. It may be mounting small-size memfs not large enough for large shuffle data. It might be better to use a unique dir here like `spark-shuffle-dir` and document that people should prepare the dir if necessary.

Eventually we should perhaps make this configurable by turning this into a helm chart?